lib/types/types.d.ts

```diff
@@ -87,3 +87,24 @@ export type CallsConfig = {
 export type Reaction = UserReactionData & {
     displayName: string;
 };
+export type CallUserState = {
+    unmuted: boolean;
+    raised_hand: number;
+};
+export type CallState = {
+    id: string;
+    start_at: number;
+    users: string[];
+    states?: CallUserState[];
```

Comment:
The following is compared to what I have on mobile (just want to double check):
states is not optional (but could be an empty [])